### DISCUSSION TITLE
Are the useraccounts-* packages still maintained?

### DISCUSSION THREAD

**Discussion by manueltimita** [meteor/meteor#12917] [Compat Packages]
I would like to start working on some of these, more urgently on `useraccounts-core`, to make sure they are compatible with the new async/await approach. Possibly making two releases, one that would work with Meteor 2.8+ and one for Meteor 3.0.

Question is, are these packages still maintained? Would anyone merge my pull requests?

Also, I would like to discuss a couple of things, like the best approach for some of the APIs.

**Comment by StorytellerCZ** [comment]
`useraccounts` packages are community packages. Direct your questions to the repositories where the code is to get a proper answer.
From my knowledge these packages has long been abandoned. There has been some compat update around 2.3 I believe, but that is about it.

**Reply by manueltimita** [reply]
Thank you. I knew they have been long abandoned, but then I saw them forked under Meteor Compat Packages and thought an update would benefit them. No probs, I will maintain them as separate forks
